test/m4/appframework/appframework_test.go

```diff
@@ -138,184 +143,185 @@ var _ = Describe("m4appfw test", func() {
 
-			// Ensure the indexers of all sites go to Ready phase
+			// Ensure the Indexers of all sites go to Ready phase
 			testenv.IndexersReady(deployment, testenvInstance, siteCount)
 
-			// Ensure cluster configured as multisite
+			// Ensure cluster configured as Multisite
 			testenv.IndexerClusterMultisiteStatus(deployment, testenvInstance, siteCount)
 
-			// Ensure SHC go to Ready phase
+			// Ensure Search Head Cluster go to Ready phase
 			testenv.SearchHeadClusterReady(deployment, testenvInstance)
 
 			// Verify RF SF is met
 			testenv.VerifyRFSFMet(deployment, testenvInstance)
 
-			// Verify MC is Ready and stays in ready state
+			// Verify Monitoring Console is Ready and stays in ready state
 			testenv.VerifyMonitoringConsoleReady(deployment, deployment.GetName(), mc, testenvInstance)
 
-			// Verify apps are downloaded by init-container for M4 SVA
-			testenvInstance.Log.Info("Verify V2 apps are downloaded by init container for M4 for apps", "version", appVersion)
+			//########## UPGRADE VERIFICATIONS ##########
+			// Verify V2 apps are downloaded on Cluster Manager and Deployer
+			testenvInstance.Log.Info(fmt.Sprintf("Verify %s apps are downloaded on Cluster Manager and Deployer", appVersion))
 			testenv.VerifyAppsDownloadedOnContainer(deployment, testenvInstance, testenvInstance.GetName(), podNames, appFileList, initContDownloadLocation)
 
-			// Verify apps downloaded by init-container for Monitoring Console
-			testenvInstance.Log.Info("Verify V2 apps are downloaded by init container for MC for apps", "POD", mcPodName, "version", appVersion)
+			// Verify V2 apps are downloaded on Monitoring Console
+			testenvInstance.Log.Info(fmt.Sprintf("Verify %s apps are downloaded on Monitoring Console pod %s", appVersion, mcPodName))
 			testenv.VerifyAppsDownloadedOnContainer(deployment, testenvInstance, testenvInstance.GetName(), []string{mcPodName}, appFileList, initContDownloadLocationMCPod)
 
 			// Verify bundle push status and compare bundle hash with previous V1 bundle
 			testenv.VerifyClusterManagerBundlePush(deployment, testenvInstance, testenvInstance.GetName(), siteCount, clusterManagerBundleHash)
 			testenv.VerifyDeployerBundlePush(deployment, testenvInstance, testenvInstance.GetName(), shReplicas)
 
-			// Verify apps are copied to location for M4 SVA
-			testenvInstance.Log.Info("Verify apps are copied to correct location based on Pod KIND for app", "version", appVersion)
+			// Verify V2 apps are copied to location for M4 SVA
+			testenvInstance.Log.Info(fmt.Sprintf("Verify %s apps are copied to correct location on M4 pods", appVersion))
 			testenv.VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), allPodNames, appListV2, true, true)
 
-			// Verify apps are copied to location for Monitoring Console
-			testenvInstance.Log.Info("Verify apps are copied to correct location for Monitoring Console", "version", appVersion)
+			// Verify V2 apps are copied to location for Monitoring Console
```

Comment:
We don't call these as "V2" apps in the comments of the original PR. Should develop branch be updated to be consistent?

Reply:
I would say it's still ok, the appVersion is present in the "testenvInstance.Log.Info" lines of that code block